tests/local/network.go

```diff
@@ -220,22 +199,20 @@ func (n *LocalNetwork) setPrimaryNetworkValues() {
 }
 
 func (n *LocalNetwork) setSubnetValues(subnetID ids.ID) {
-	subnetDetails, ok := n.manager.GetSubnet(subnetID)
-	Expect(ok).Should(BeTrue())
-	blockchainID := subnetDetails.BlockchainID
+	subnet := n.tmpnet.Subnets[slices.IndexFunc(
```

Comment:
It looks like in all cases `setSubnetValues` is called, we have a `*tmpnet.Subnet` and we pass the `.SubnetID` of it, only to use that ID to look up the `*tmpnet.Subnet` at the start here. Lets just pass the `*tmpnet.Subnet` directly.

Reply:
good call, thank you. addressed in https://github.com/ava-labs/teleporter/pull/401/commits/6f790165bb4257b5fa4eed916e3366f4f87c98e3